phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -290,7 +290,7 @@ private void openConnection() throws SQLException {
     @Override
     public HTableInterface getTable(byte[] tableName) throws SQLException {
         try {
-            return HBaseFactoryProvider.getHTableFactory().getTable(tableName, connection, null);
+            return HBaseFactoryProvider.getHTableFactory().getTable(tableName, connection, getExecutor());
```

Comment:
Revert this change please - it was part of our experiment to get the RpcScheduler set on the client-side.